control-plane/bfruntime.cpp

```diff
@@ -292,7 +292,8 @@ BFRuntimeGenerator::addKeyField(Util::JsonArray* dataJson, P4Id id, cstring name
 BFRuntimeGenerator::initTableJson(const std::string& name,
         P4Id id, cstring tableType, int64_t size, Util::JsonArray* annotations) {
     auto* tableJson = new Util::JsonObject();
-    tableJson->emplace("name", name);
+    const std::string tableName = "pipe." + name;
```

Comment:
Here as well, this is incorrect. 'pipe' is the name of the pipeline, not a fixed prefix.